source/src/Internal.cc

```diff
@@ -0,0 +1,108 @@
+/*
+ *
+ * Internal.cc source template automatically generated by a class generator
+ * Creation date : lun. ao�t 17 2015
+ *
+ * This file is part of DQM4HEP libraries.
+ *
+ * DQM4HEP is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * based upon these libraries are permitted. Any copy of these libraries
+ * must include this copyright notice.
+ *
+ * DQM4HEP is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with DQM4HEP.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @author Remi Ete
+ * @copyright CNRS , IPNL
+ */
+
+// -- dqm4hep headers
+#include <dqm4hep/Internal.h>
+
+#include <cstring>
+
+namespace dqm4hep {
+
+  namespace core {
+    
+    // only for unix systems
```

Comment:
Maybe add an #ifdef __linux__ to make sure no other os tries to call it (like APPLE...)

Reply:
I implemented a fix for that here : 
https://github.com/DQM4HEP/dqm4hep/pull/16

Reply:
Done and tested the fix on OSX.  Waiting for travis now